src/app/student-login/student-login.component.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../variables";
+
+
+.page{
+    background-repeat: no-repeat;
+    background-image: url('/assets/images/large-display/hill-compressed.jpg');
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
Since we will probably be using this type of background in many places we should move the parts of this that set the background image and position it to a new class in the global styles.

Reply:
Do we also want to be setting the height and width in this class? At the moment I've just moved the three background styles, but it doesn't show the full image when applied to elements that are way wider than they are tall or vice versa (see [the session test page](https://buzz-about-gg557td96.vercel.app/session-page-test))

Reply:
Yeah I think the height and width should be left up to the individual component. The way you did it seems right. One if the ideas of CSS is that you can use multiple classes like this where each one does a small thing.